packages/smart-contracts/src/lib/artifacts/BatchPayments/index.ts

```diff
@@ -61,10 +61,14 @@ export const batchPaymentsArtifact = new ContractArtifact<BatchPayments>(
           address: '0x0DD57FFe83a53bCbd657e234B16A3e74fEDb8fBA',
           creationBlockNumber: 35498688,
         },
-        zkSyncEra: {
+        zksyncera: {
           address: '0x0C41700ee1B363DB2ebC1a985f65cAf6eC4b1023',
           creationBlockNumber: 19545614,
         },
+        zksynceratestnet: {
+          address: '0x276A92F78aA527b8e157B0E47cEB63b4239dfa0b',
```

Comment:
Why does the zkSync Era Goerli explorer show the contract name "BatchConversionPayments" instead of "BatchPayments"?
![image](https://github.com/RequestNetwork/requestNetwork/assets/2530913/25462f9b-6532-4801-b856-7662cfa39227)